### PR TITLE
Add contribution guide

### PR DESCRIPTION
**Pull request by saschatimme**
I think that's a acceptable starting point, but I would love to have some feedback :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Contributing guide
+Great that you consider to contribute to the Bucklescript bindings for React Native. **Every contribution is greatly appreciated!**
+There are many ways to contribute, besides implementing new features and fixing bus, you can give advice in issues, create and improve documentation, etc.
```
Comment (wokalski) [inline CONTRIBUTING.md]: s/bus/bugs

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+## I want to contribute a missing component or API
+That's awesome!. If you have a specific one in mind go for it, otherwise just pick one of your choice.
+If you expect that it would take you a while (which is no problem) you can open a PR with your working progress to let everybody know you are working on it.
+To add component (or API) you should do the following steps:
+* Implement your component in a file `myComponentRe.re`
+* Add an interface `myComponentRe.rei` and think about which things should be exposed.
+* Add your new component to `reactNative.re` following the existing components / APIs.
+
+Some things to keep in mind:
+* We strive for more heavy bindings, i.e. we try to hide things like `Js.t` objects. Sometimes this is not possible
```
Comment (wokalski) [inline CONTRIBUTING.md]: `.` missing at the end.

**Comment by kennetpostigo** [comment]
@saschatimme this looks great!

**Comment by wokalski** [comment]
This is awesomest

**Pull request merged**